nav_msgs/Path path
---
bool          is_collision_course